### DISCUSSION TITLE
Standalone vs Cluster mode limitations

### DISCUSSION THREAD

**Discussion by nairan-deshaw** [milvus-io/milvus#35697] [Q&A and General discussion] [answered]
Hi, we would like to know some numbers on the amount of data that standalone and cluster modes support. Another consideration is what is the resource footprint difference between the two modes. 
The use case is to know at what number of vectors do we want to switch to a cluster mode.

**Comment by yhmo** [comment, marked as answer]
There is a guideline here for how to choose the proper deployment: https://milvus.io/docs/install-overview.md

A quick guideline:
- Milvus Lite is recommended for smaller datasets, up to a few million vectors.
- Milvus Standalone is suitable for medium-sized datasets, scaling up to 100 million vectors.
- Milvus Distributed is designed for large-scale deployments, capable of handling datasets from 100 million up to tens of billions of vectors.

![select-deployment-option](https://github.com/user-attachments/assets/3fd60f14-aa61-4878-b8c5-e8fb40da829b)

And there is a tool to help estimate the required resources of a milvus cluster: https://milvus.io/tools/sizing

**Comment by nairan-deshaw** [comment]
Thanks @yhmo!
